advocacy_docs/edb-postgres-ai/ai-accelerator/retrievers/concepts.mdx

```diff
@@ -1,66 +1,66 @@
+Retrievers always need a *vector table* to store the vector embeddings that belong to the source data. We use a model with *encoder* capabilities to generate the vectors for the source data.
 
-Each model has a different "dimensionality" i.e. number of dimensions of the vector (the vector [1, 2, 2] has 3 dimensions). We use pgvector "vector" column types to store the embeddings. And those have a fixed number of dimensions. For this reason, the encoder interface on the models returns how many dimensions the vectors will have. 
+Each model has a different *dimensionality*, that is, the number of dimensions of the vector. (The vector [1, 2, 2] has 3 dimensions.) Pipelines uses PGvector *vector* column types to store the embeddings. Those have a fixed number of dimensions. For this reason, the encoder interface on the models returns the number of dimensions the vectors will have. 
 
-We create the vector table, or support using an existing one. The vector column dimensionality must fit the model.
+Pipelines creates the vector table and also supports using an existing one. The vector column dimensionality must fit the model.
 
 ### Consistency with source data
 
-Embeddings must be in-sync with the source data in order to retrieve correct and consistent data. In the case of the table data source, we use triggers in postgres to be notified about changes on the source table.
+To retrieve correct and consistent data, embeddings must be in sync with the source data. In the case of the table data source, Pipelines uses triggers in Postgres to be notified about changes on the source table.
 
-If the source table already has data/rows at the time where the retriever is created, then a manual "bulk embedding" call must be made.
+If the source table already has data/rows when the retriever is created, then a manual bulk embedding call must be made.
 
-Auto embedding can be activated to keep the embeddings in sync going forward.
+To keep the embeddings in sync going forward, you can activate auto-embedding.
 
-### Bulk & Auto embedding
+### Bulk embedding and auto-embedding
 
-Bulk embedding generates the embeddings for all the existing data in a source table. Use this to initialize the retriever if the source table is not empty.
+Bulk embedding generates the embeddings for all the existing data in a source table. Use it to initialize the retriever if the source table isn't empty.
 
 !!! Note
-Bulk embedding does not "sync" the data; it’s best to run this on an empty vector table.
+Bulk embedding doesn't sync the data. We recommend running it on an empty vector table.
 !!!
 
-Auto embedding can be manually enabled. It creates triggers in PG to keep the embeddings up to data if source data is added, updated or removed.
+Auto-embedding can be manually enabled. It creates triggers in Postgres to keep the embeddings up to data if source data is added, updated, or removed.
```

Comment:
We generally don't refer to Postgres in this context as PG but spell it out, assuming that's what's meant here.